examples/deadline/All-In-AWS-Infrastructure-Basic/python/README.md

```diff
@@ -123,12 +123,36 @@ These instructions assume that your working directory is `examples/deadline/All-
     ```python
     self.alarm_email_address: Optional[str] = 'username@yourdomain.com'
     ```
-15. Deploy all the stacks in the sample app:
+15. Deadline Secrets Management is a feature used to encrypt certain values in the database that need to be kept secret. Additional documentation about the feature and how it works in the RFDK can be found in the [RFDK README](../../../../packages/aws-rfdk/lib/deadline/README.md). By default, Deadline Secrets Management is enabled, but it can be disabled by changing the `enable_secrets_management` variable in `package/config.py`.
+
+    ```python
+    self.enable_secrets_management: bool = False
+    ```
+
+16. When you are using Deadline Secrets Management you can define your own admin credentials by creating a Secret in AWS SecretsManager in the following format:
+
+    ```json
+        {
+            "username": <admin user name>,
+            "password": <admin user password>,
```

Comment:
I just realized the reason these values are highlighted red in the review is because they are missing quotes around them, making them invalid JSON. The rich diff also shows they are highlighted in red, so I think maybe we should fix it now to make sure that it won't look like that when it's committed:
```suggestion
            "username": "<admin user name>",
            "password": "<admin user password>",
```